src/zeit/content/volume/browser/toc.py

```diff
@@ -0,0 +1,413 @@
+        ids_as_string = config.get('toc-product-ids')
+        if not ids_as_string:
+            ids_as_string = toc_config.PRODUCT_IDS
+        return [product_id.strip() for product_id in ids_as_string.split(' ')]
+
+    def _get_all_paths_for_product_ids(self, product_ids):
+        """
+        Creates a list of unix-paths to all given products
+        :param product_ids: [str]
+        :return: [str]
+        """
+        product_dir_names = [self._replace_product_id_by_its_dirname(prod_id)
+                             for prod_id in product_ids]
+        # Volumes <10 would lead to wrong paths like YEAR/1 instead of YEAR/01
+        volume_string = '%02d' % self.context.volume
+        # You need the XML prefix here
+        prefix = 'http://xml.zeit.de'
+        return [posixpath.join(*[str(e) for e in
+                [prefix, dir_name, self.context.year, volume_string, '']])
+                for dir_name in product_dir_names]
+
+    def _replace_product_id_by_its_dirname(self, product_id):
+        """
+        :param product_id: str
+        :return: str
+        """
+        return self.PRODUCT_ID_DIR_NAME_EXCEPTIONS.get(product_id, product_id)
+
+    def list_relevant_ressort_folders(self, path):
+        """
+        :param path: path to product for the volume
+        :return: [('foldername', zeit.cms.repository.folder.Folder), ...]
+        """
+        try:
+            product_folder = \
+                zeit.cms.interfaces.ICMSContent(self.connector[path])
+            return [item for item in product_folder.items()
+                    if self._is_relevant_folder_item(item)]
+        except KeyError:
+            return []
+
+    def _is_relevant_folder_item(self, item):
+        return self.excluder.is_relevant_folder(item[0]) \
+               and IFolder.providedBy(item[1])
+
+    def _get_all_article_elements(self, ressort_folder):
+        """
+        Get all DAV Server paths to article files in path.
+        :param ressort_folder:
+        :return: [lxml.etree Article element, ...]
+        """
+        return [resource.xml for _, resource in ressort_folder.items()
+                if IArticle.providedBy(resource)]
+
+    def _create_toc_element(self, article_element):
+        """
+        :param article_element: lxml.etree Article element
+        :return: {'page': str, 'author': str, 'title': str, 'teaser': str}
+        """
+        return self._get_metadata_from_article_xml(article_element) \
+            if self.excluder.is_relevant(article_element) else None
+
+    def _get_metadata_from_article_xml(self, atricle_tree):
+        """
+        Get all relevant normalized metadata from article xml tree.
+        :param atricle_tree: lxml.etree Element
+        :return: {'page': str, 'author': str, 'title': str, 'teaser': str}
+        """
+        xpaths = {
+            'title': "body/title/text()",
+            'page': "//attribute[@name='page']/text()",
```

Comment:
Mache ich. Das Problem taucht in ähnlicher Weise auch für Produkte wie Christ und Welt auf, deren page-property auch nicht richtig geparsed werden kann. So z.B. bei 
Geht man nach /repository/2016/25/ bekommt man:
2016-11-18 12:08:00,083 WARNI zeit.cms.content.dav Could not parse DAV property value 'CW1' for Article.page at http://xml.zeit.de/2016/25/Neudeck-box-1 [ValueError: ("invalid literal for int() with base 10: 'CW1'",)]. Using default None instead.